src/libraries/Microsoft.Bcl.Numerics/src/Microsoft.Bcl.Numerics.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
How about also targeting .NET Standard 2.1?

Reply:
.NET Standard 2.1 would be able to use the type forwards as well if its supported.

Reply:
Good idea @teo-tsirpanis. We are now explicitly targeting .NET Standard 2.1 so we can correctly forward as needed.